jit/v2/JIT_v2.0.0.yaml

```diff
@@ -327,7 +423,7 @@ paths:
       operationId: get-port-call
       summary: Retrieves a list of Port Calls
       description: |
-        Description...
+        Fetches a list of **Port Calls** that match the filter criterias. If only a specific **Port Call** is requested then it is possible to filter the result by `portCallID` which will return an array with at most a single object (the array can also be empty in case the `portCallID` is not linked to any **Port Call** known by the consumer).
```

Comment:
Retrieves a list of **Port Calls** that match the specified filter criteria. To retrieve details for a specific **Port Call**, use the `portCallID` filter. This will return an array containing at most a single object. The response will return an empty array if no **Port Call** known by the consumer is linked to the provided `portCallID`.